### DISCUSSION TITLE
GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by safaxx** [github/india#352] [Students]
Hello All, I wanted to know if anyone has Claimed/Received any information regarding the GitHub Student Developer Pack yet?

**Comment by divyavemasani2910** [comment]
No not yet...i gave request but there is no response

**Reply by TheSuranaverse** [reply]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together.
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/jPv7hydk
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together.
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/jPv7hydk
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏
